### PR TITLE
Update dependencies (libcosmic + iced 0.13), add dbus-config feature

### PR DESCRIPTION
**Pull request by sungsphinx**

### CODE REVIEW

**Review by ahoneybun — APPROVED** [review]
LGTM

File: Cargo.toml [hunk 1]
```diff
@@ -29,8 +29,9 @@ features = ["tokio", "winit"]
 vergen = { version = "8", features = ["git", "gitcl"] }
 
 [features]
-default = ["wgpu"]
+default = ["dbus-config", "wgpu"]
 wgpu = ["libcosmic/wgpu"]
+dbus-config = ["libcosmic/dbus-config"]
```
Comment (lilyinstarlight) [inline Cargo.toml:34]: It looks like Cargo.lock was not regenerated after the `dbus-config` feature was added. This is making cargo attempt to re-lock dependencies whenever an attempt is made to build the project and breaks downstream tooling

See the change to Cargo.lock when running `cargo metadata` after checking out this repo:

```diff
diff --git a/Cargo.lock b/Cargo.lock
index 123c154..7caa87b 100644
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -1055,7 +1055,9 @@ source = "git+https://github.com/pop-os/libcosmic.git#a73e91a12bbc4318b855d7ab10
 dependencies = [
  "atomicwrites",
  "cosmic-config-derive",
+ "cosmic-settings-daemon",
  "dirs",
+ "futures-util",
  "iced_futures",
  "known-folders",
  "notify",
@@ -1065,6 +1067,7 @@ dependencies = [
  "tokio",
  "tracing",
  "xdg",
+ "zbus 4.4.0",
 ]
 
 [[package]]
@@ -1076,6 +1079,14 @@ dependencies = [
  "syn 1.0.109",
 ]
 
+[[package]]
+name = "cosmic-settings-daemon"
+version = "0.1.0"
+source = "git+https://github.com/pop-os/dbus-settings-bindings#931f5db558bf3fcb572ff4e18f7f1618a7430046"
+dependencies = [
+ "zbus 4.4.0",
+]
+
 [[package]]
 name = "cosmic-text"
 version = "0.12.1"
@@ -2850,6 +2861,7 @@ dependencies = [
  "ashpd 0.9.2",
  "chrono",
  "cosmic-config",
+ "cosmic-settings-daemon",
  "cosmic-theme",
  "css-color",
  "derive_setters",
```

Comment (sungsphinx) [reply Cargo.toml:34]: Hi @lilyinstarlight, I have applied that diff and pushed it to main, sorry for the inconvenience!

Comment (lilyinstarlight) [reply Cargo.toml:34]: No worries, thanks so much for the prompt response! :)

**Pull request merged**